<launch>
    <arg name="legacy_packets" default="false" />
    <param name="use_legacy_packets" type="boolean" value="$(arg legacy_packets)" />

    <arg name="robot_output_target" default="grsim" />
    <param name="robot_output_target" type="string" value="$(arg robot_output_target)" />

    <arg name="should_normalize_field" default="true" />
    <param name="normalize_field" type="boolean" value="$(arg should_normalize_field)" />

    <arg name="our_field_side" default="left" />
    <param name="our_side" type="string" value="$(arg our_field_side)" />

    <node pkg="roboteam_world" type="filtered_world" name="world"/>
    <node pkg="roboteam_vision" type="roboteam_vision" name="vision"/>
    <param name="serialOut" type="string" value="/dev/ttyACM0" />
    <node pkg="roboteam_robothub" type="roboteam_robothub" name="robothub" output="screen"/>
</launch>
